utils/test/testUtilsMouseEvents.ts

```diff
@@ -44,4 +47,57 @@ function createNormalizedMouseEvent(
   };
 }
 
-export { createNormalizedMouseEvent };
+/**
+ * Asynchronously dispatches a mouse down followed by a mouse up on the given element.
+ * Since mouse down events are performed on a timeout to detect potential
+ * double clicks, the mouse up event is not triggered until the mouse down
+ * event has been processed. An optional callback is invoked after the mouse
+ * down is triggered but before the mouse up. An optional callback is invoked
+ * after the mouse up has fired.
+ *
+ * @param element the element to dispatch to
+ * @param mouseDownEvent the mouse down event to dispatch
+ * @param mouseUpEvent the mouse up event to dispatch
+ * @param betweenDownAndUpCallback optional callback between the down and up
+ * @param afterDownAndUpCallback optional callback after the up
+ * @returns a Promise for the eventual completion of the mouse down and up
+ */
+function doMouseDownAndUp(
```

Comment:
can we rename this function to `performMouseDownAndUp`?

Reply:
as you wish